sky/clouds/service_catalog/data_fetchers/fetch_lambda_cloud.py

```diff
@@ -0,0 +1,119 @@
+"""A script that generates the Lambda Cloud catalog.
+
+Usage:
+    python fetch_lambda_cloud.py [-h] [--api-key API_KEY]
+                                 [--api-key-path API_KEY_PATH]
+
+If neither --api-key nor --api-key-path are provided, this script will parse
+`~/.lambda/lambda_keys` to look for Lambda api key.
+"""
+import argparse
+import csv
+import json
+import os
+import requests
+
+ENDPOINT = 'https://cloud.lambdalabs.com/api/v1/instance-types'
+LAMBDA_KEYS_PATH = os.path.expanduser('~/.lambda_cloud/lambda_keys')
+
+# This is the list that Lambda Labs gave us.
+regions = [
+    'australia-southeast-1',
+    'europe-central-1',
+    'asia-south-1',
+    'me-west-1',
+    'europe-south-1',
+    'asia-northeast-1',
+    'asia-northeast-2',
+    'us-east-1',
+    'us-west-2',
+    'us-west-1',
+    'us-south-1',
+]
+
+gpu_to_memory = {
```

Comment:
ditto

Reply:
Can you add a link from which you get this data?

Reply:
Added!